content/documentation/edge/features/index.adoc

```diff
@@ -17,20 +18,27 @@ type: "features"
 :icons: font
 :imagesdir: /images/documentation/features/
 
-== Overview
-
 Kiali helps you define, validate, and observe the connections and microservices of your Istio service mesh.
 
-Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
+Kiali works with Istio in Kubernetes distributions. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
 
-icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali also includes Jaeger Tracing to provide distributed tracing by default.
+Kiali also helps to operate the Service Mesh providing Wizards to apply common traffic patterns an generate automatically Istio configuration.
```

Comment:
operate --> manage (optional)
an --> and (required)
generate automatically --> automatically generate (optional)